tests/test_commands.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import sys
+
+from .test_plugins.test_plugins import function_like_plugin
+
+try:
+    import importlib.metadata as importlib_metadata
+except ModuleNotFoundError:
+    import importlib_metadata
+
+import manim
+
+
+def call_command(command, cwd=None, env=None):
+    a = subprocess.run(
+        command,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=True,
+        text=True,
+        cwd=cwd,
+        env=env,
+    )
+    return a
+
+
+def test_manim_version_from_command_line():
+    a = call_command(
+        [
+            sys.executable,
+            "-m",
+            "manim",
+            "--version",
+        ]
+    )
+    version = importlib_metadata.version("manim")
+    assert version in a.stdout
+    assert a.stdout.strip() == f"Manim Community v{version}"
+    assert manim.__version__ == version
```

Comment:
This assertion will always be true because `manim.__version__` is set the same way as you set `version` above. Instead, there is a already a test [here](https://github.com/ManimCommunity/manim/blob/master/tests/test_version.py) that _should_ be comparing `__version__` to the version in `pyproject.toml`.

Reply:
oh! didn't notice that, in that case, I will remove it.